### PR TITLE
Bulk re-enabling of default_off="refused" rulesets.

### PR DESCRIPTION
**Pull request by bardiharborow**

### CODE REVIEW

File: src/chrome/content/rules/MPort.ua.xml [hunk 1]
```diff
@@ -2,17 +2,18 @@
...
 
 
-	<!--	Not secured by server:
-					-->
-	<!--securecookie host="^\.ua$" name="^bm_lang$" /-->
+	<securecookie host=".+" name=".+" />
 
 
+	<rule from="^http://www\.mport\.ua/"
+		to="https://mport.ua/" />
```
Comment (J0WI) [inline src/chrome/content/rules/MPort.ua.xml]: Please add/update the comments for such redirects.

Comment (bardiharborow) [reply src/chrome/content/rules/MPort.ua.xml]: @J0WI oops, fixed.

File: src/chrome/content/rules/Rawstudio.org.xml [hunk 2]
```diff
@@ -1,25 +1,21 @@
 <!--
 	Nonfunctional subdomains:
 
-		- (www.)? ¹
```
Comment (J0WI) [inline src/chrome/content/rules/Rawstudio.org.xml]: Same here

File: src/chrome/content/rules/Utopia.xml [hunk 3]
```diff
@@ -1,8 +1,4 @@
-<!--
-	(www.)?: Dropped
```
Comment (J0WI) [inline src/chrome/content/rules/Utopia.xml]: and here

File: src/chrome/content/rules/Vocaroo.com.xml [hunk 4]
```diff
@@ -1,9 +1,12 @@
-<ruleset name="Vocaroo.com" default_off="refused">
+<ruleset name="Vocaroo.com">
 
 	<target host="vocaroo.com" />
 	<target host="www.vocaroo.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Vocaroo.com.xml]: and here

**Comment by J0WI** [comment]
@bardiharborow could you fix those minor issues?

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**